doc/api/crypto.md

```diff
@@ -2082,6 +2082,17 @@ encryption mechanism, PEM-level encryption is not supported when encrypting
 a PKCS#8 key. See [RFC 5208][] for PKCS#8 encryption and [RFC 1421][] for
 PKCS#1 and SEC1 encryption.
 
+### `keyObject.equals(otherKeyObject)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `otherKeyObject`: {KeyObject} A `KeyObject` with which to
+  compare `keyObject`.
+* Returns: {boolean} `true` or `false` depending on whether the
+  keys have exactly the same type, value, and parameters.
```

Comment:
I'd mention that the comparison is not constant time.

Reply:
It is for symmetric keys as per @tniessen's request.

Reply:
Nevertheless, would you suggest a wording?

Reply:
"This method is not [constant time](https://en.wikipedia.org/wiki/Timing_attack)." :-)